docs/kb/logging.md

```diff
@@ -183,14 +185,88 @@ the replicating process will receive this information when it connects.
 
 On start-up, the replicating process connects to the logging process, gets the log filename and record count, opens the log file, plays back that count of records from the log file, and continues to receive updates via [TCP/IP](../basics/ipc.md). Each record is executed via `value`.
 
-If the replicating process loses its connection to the logging process, you can detect that with `.z.pc`. To resubscribe to the logging process, restart the replicating process.
+If the replicating process loses its connection to the logging process, you can detect that with [`.z.pc`](../ref/dotz.md#zpc-close). To resubscribe to the logging process, restart the replicating process.
 
 Currently, only a single replicating process can subscribe to the primary process. If another kdb+ process attempts to replicate from the primary, the previous replicating process will no longer receive updates. If you need multiple replicating processes, you might like to consider [kdb+tick](../learn/startingkdb/tick.md).
 
 
+## Manual handling
 
-## Replaying log files
+### Overview
+
+Function calls and the contents of the their parameters can be recorded to a log file, which can then be replayed by a process. This is often used for data recovery.
+
+This technique is often used to allow a developer more control over log file naming conventions, what to log, log file locations and the ability to add logic around the log file lifecycle.
+
+### Creating a log file
+
+A log file can be initialized using [`set`](../ref/get.md#set).
+
+```q
+q)logfile:hsym `$"qlog";
+q)logfile set ();
+q)logfilehandle:hopen logfile;
+```
+
+!!!note "`logfile set ();` is equivalent to `.[logfile;();:;()];`"
+
+A check for a pre-existing log file can be achieved using [`get`](../ref/get.md#get). 
+The script can now be written to initialize a new log file if it does not exist, otherwise it will opened for appending.
```

Comment:
```suggestion
1. Check for pre-existing log files by using [`get`](../ref/get.md#get). 
1. If a new log file does not exist, the script can now be written to initialize it, otherwise it is opened for appending.
```